docs/charts.md

```diff
@@ -479,7 +479,7 @@ Furthermore, A is dependent on chart B that creates objects
 - replicaset "B-ReplicaSet"
 - service "B-Service"
 
-After installation/upgrade of chart A a single Helm release is created/modified. The release will
```

Comment:
this is not a typo. Perhaps the grammar should be corrected instead, adding a comma after "chart A". e.g.

> After installation/upgrade of chart A, a single Helm release is created/modified.

Reply:
I have corrected the grammar of this part.